aikau/src/main/resources/alfresco/layout/UploadContainer.js

```diff
@@ -100,18 +104,40 @@ define(["dojo/_base/declare",
        * @instance
        */
       setDndHighlightDimensions: function alfresco_documentlibrary__AlfDndDocumentUploadMixin__setDndHighlightDimensions() {
-         if (this.proxyDragAndDropNode && query(this.proxyDragAndDropNode).length === 1)
+
+         // proxyDragAndDropNode can be an array of strings or a comma-separated list
+         if (this.proxyDragAndDropNode && !(this.proxyDragAndDropNode instanceof Array))
```

Comment:
Use of instanceof Array isn't actually sufficient to determine if something is actually an array (see http://web.mit.edu/jwalden/www/isArray.html). We provide the alfresco/core/ObjectTypeUtils isArray function for accurately determining if something is an array or not. This uses the Dojo polyfill for browsers that don't support Array.isArray (see https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)